infra/src/Pos/Infra/Communication/Types/Relay.hs

```diff
@@ -65,6 +66,10 @@ data DataMsg contents = DataMsg
     { dmContents :: !contents
     } deriving (Generic, Show, Eq)
 
+instance Bi (DataMsg TxMsgContents) where
```

Comment:
Wouldn't it make more sense if this `Bi` instance lived in `Pos.Txp.Network.Types` where `TxMsgContents` is defined?